src/messaging/ReliableMessageContext.h

```diff
@@ -169,14 +166,17 @@ class ReliableMessageContext
         /// When set, we have had Close() or Abort() called on us already.
         kFlagClosed = (1u << 7),
 
-        /// When set, signifies that the exchange is requesting Sleepy End Device active mode.
-        kFlagActiveMode = (1u << 8),
-
         /// When set, signifies that the exchange created sorely for replying a StandaloneAck
-        kFlagEphemeralExchange = (1u << 9),
+        kFlagEphemeralExchange = (1u << 8),
 
         /// When set, ignore session being released, because we are releasing it ourselves.
-        kFlagIgnoreSessionRelease = (1u << 10),
+        kFlagIgnoreSessionRelease = (1u << 9),
+
+        // This flag is used to determine if the peer (receiver) should be active or not
+        // When set, sender knows it has received at least one application-level message
+        // from the peer and can assume the peer (receiver) is active
```

Comment:
Please put a period at end of each sentence. Helps reading the comments.

Reply:
it would be good grammar also on my part